<launch>

  <!-- Run a python script to the send a service call to gazebo_ros to spawn a grasp objects -->
  <node name="spawn_threePrism" pkg="gazebo_ros" type="spawn_model" args="-file $(find robot_grasp)/models/grasp_object/threePrism/threePrism.sdf -sdf -model threePrism -x 0.262906  -y -0.176547 -z 0.731734" respawn="false" output="screen" />
  <!-- node name="spawn_sixPrism" pkg="gazebo_ros" type="spawn_model" args="-file $(find robot_grasp)/models/grasp_object/sixPrism/sixPrism.sdf -sdf -model sixPrism  -x 0.374627 -y -0.182299 -z 0.744235" respawn="false" output="screen" /> -->
</launch>
